05/17/2024 16:22:36 - INFO - __main__ - Distributed environment: NO
Num processes: 1
Process index: 0
Local process index: 0
Device: cuda

Mixed precision type: no

You are using a model of type clip_text_model to instantiate a model of type . This is not supported for all configurations of models and can yield errors.
{'variance_type', 'thresholding', 'dynamic_thresholding_ratio', 'rescale_betas_zero_snr', 'clip_sample_range'} was not found in config. Values will be initialized to default values.
{'latents_mean', 'latents_std'} was not found in config. Values will be initialized to default values.
{'reverse_transformer_layers_per_block', 'dropout', 'attention_type'} was not found in config. Values will be initialized to default values.
05/17/2024 16:23:00 - INFO - __main__ - ***** Running training *****
05/17/2024 16:23:00 - INFO - __main__ -   Num examples = 200
05/17/2024 16:23:00 - INFO - __main__ -   Num batches each epoch = 200
05/17/2024 16:23:00 - INFO - __main__ -   Num Epochs = 4
05/17/2024 16:23:00 - INFO - __main__ -   Instantaneous batch size per device = 1
05/17/2024 16:23:00 - INFO - __main__ -   Total train batch size (w. parallel, distributed & accumulation) = 1
05/17/2024 16:23:00 - INFO - __main__ -   Gradient Accumulation steps = 1
05/17/2024 16:23:00 - INFO - __main__ -   Total optimization steps = 800
Steps:   0%|          | 0/800 [00:00<?, ?it/s]Traceback (most recent call last):
  File "/net/tscratch/people/plgfilipeckijan/zzsn-project/train_dreambooth_sdxl.py", line 1443, in <module>
    main(args)
  File "/net/tscratch/people/plgfilipeckijan/zzsn-project/train_dreambooth_sdxl.py", line 1273, in main
    model_pred = unet(
  File "/net/tscratch/people/plgfilipeckijan/.conda/envs/zzsn_project_env/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/net/tscratch/people/plgfilipeckijan/.conda/envs/zzsn_project_env/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
  File "/net/tscratch/people/plgfilipeckijan/.conda/envs/zzsn_project_env/lib/python3.9/site-packages/diffusers/models/unets/unet_2d_condition.py", line 1152, in forward
    aug_emb = self.get_aug_embed(
  File "/net/tscratch/people/plgfilipeckijan/.conda/envs/zzsn_project_env/lib/python3.9/site-packages/diffusers/models/unets/unet_2d_condition.py", line 973, in get_aug_embed
    if "text_embeds" not in added_cond_kwargs:
TypeError: argument of type 'NoneType' is not iterable
Steps:   0%|          | 0/800 [00:01<?, ?it/s]
Traceback (most recent call last):
  File "/net/tscratch/people/plgfilipeckijan/.conda/envs/zzsn_project_env/bin/accelerate", line 8, in <module>
    sys.exit(main())
  File "/net/tscratch/people/plgfilipeckijan/.conda/envs/zzsn_project_env/lib/python3.9/site-packages/accelerate/commands/accelerate_cli.py", line 46, in main
    args.func(args)
  File "/net/tscratch/people/plgfilipeckijan/.conda/envs/zzsn_project_env/lib/python3.9/site-packages/accelerate/commands/launch.py", line 1075, in launch_command
    simple_launcher(args)
  File "/net/tscratch/people/plgfilipeckijan/.conda/envs/zzsn_project_env/lib/python3.9/site-packages/accelerate/commands/launch.py", line 681, in simple_launcher
    raise subprocess.CalledProcessError(returncode=process.returncode, cmd=cmd)
subprocess.CalledProcessError: Command '['/net/tscratch/people/plgfilipeckijan/.conda/envs/zzsn_project_env/bin/python', 'train_dreambooth_sdxl.py', '--pretrained_model_name_or_path=stabilityai/stable-diffusion-xl-base-1.0', '--instance_data_dir=./dataset/dreambooth-main/dataset/dog6', '--class_data_dir=./dataset/dreambooth_generated/dog', '--output_dir=./models/dreambooth_sdxl_dog6', '--with_prior_preservation', '--prior_loss_weight=1.0', '--instance_prompt=a sks dog', '--class_prompt=a dog', '--resolution=512', '--train_batch_size=1', '--gradient_accumulation_steps=1', '--learning_rate=5e-6', '--lr_scheduler=constant', '--lr_warmup_steps=0', '--num_class_images=200', '--max_train_steps=800']' returned non-zero exit status 1.
/var/spool/slurmd/job713620/slurm_script: line 40: --logging_dir=./logs/tensorboard_logs: No such file or directory
/var/spool/slurmd/job713620/slurm_script: line 41: --report_to=tensorboard: command not found
